### ISSUE TITLE
CLUS scenario tool - upload shapefile error "not valid"

### ISSUE THREAD

**Issue by TylerMuhly** [bcgov/clus#67]
I was trying to upload a shapefile in the CLUS scenario app, and got an error that the shapefile was not valid

**To Reproduce**
1. Go to scenario tool app
2. Click on "Click to Upload" under Upload Shapefile
3. Navigate to a shapefile and highlight.shp, .dbf, .shx and .prj files
4. Get error: "Shapefile not valid"

The shapefile should upload without error and be visible on the map

**Comment by TylerMuhly** [comment]
I ran the app on my local CPU to test the error. There appeared to be two problems with some syntax in two existing functions in the server:
st_transform() # line 220
bbox () # line 664

In the former, it seems CRS() function is deprecated; now changed to crs =
In the latter it appears bbox() is deprecated; now changed to st_bbox()

these updates fixed the error on my local CPU. Code was merged with Master.

Only remaining taks is to upload to virtual machine and test it.
